src/components/views/auth/InteractiveAuthEntryComponents.js

```diff
@@ -190,14 +190,21 @@ export const RecaptchaAuthEntry = createReactClass({
             return <Loader />;
         }
 
+        let errorText = this.props.errorText;
+
         const CaptchaForm = sdk.getComponent("views.auth.CaptchaForm");
-        const sitePublicKey = this.props.stageParams.public_key;
+        let sitePublicKey;
+        if (!this.props.stageParams || !this.props.stageParams.public_key) {
+            errorText = _t("Missing captcha public key - check your server configuration");
```

Comment:
Maybe something like the following, to clarify who can take action?

```suggestion
            errorText = _t("Missing captcha public key in homeserver configuration. Please report this to your homeserver administrator.");
```